Unit-01/migration/migrations/versions/0e99defa5cf4_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
```

Comment:
Your migrations don't match your model here - you either need another migration, or this migration needs to do more (e.g. create the snacks table). Right now if someone tried to fork and clone this project and run migrations, they'd get an error.